cmd/crossplane/rbac/rbac.go

```diff
@@ -53,7 +53,7 @@ var KongVars = kong.Vars{
 			ManagementPolicyBasic,
 		},
 		", "),
-	"default_registry": name.DefaultRegistry,
+	"rbac_default_registry": xpkg.DefaultRegistry,
```

Comment:
Did this need to be renamed, or were you just going for something more explicit? (Just curious.)

Reply:
actually this change was needed for the RBAC manager to be able to have its own default too. Before this change, 
The RBAC command was using the exactly same variable name (conflict) as the Core command was. So when we tell the kong cli package to parse all the vars, core is the last one listed and I think that makes it win the conflict, forcing RBAC manager to use Core's default.

This variable name change avoids this conflict and lets RBAC have its own default, as was originally intended I believe.